### PR TITLE
improve test coverage

### LINKED ISSUE #43: Improve test coverage
currently 82% - aim for acceptable coverage

Step1: pairing for 1-2 hours to analyze and then flesh out action further.

### PR DESCRIPTION
**Pull request by peetucket**
fixes #43

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -4,9 +4,13 @@ def new
     type = params[:type]
     druid = params[:druid]
     targets = params[:targets]
+    raise 'missing required params' unless type && druid && targets
```
Comment (mejackreed) [inline app/controllers/items_controller.rb]: Why not just use Rails strong params here?

Comment (peetucket) [inline app/controllers/items_controller.rb]: to do validation?  the class that is being passed the params isn't activerecord though

Comment (mejackreed) [inline app/controllers/items_controller.rb]: Yeah, I just thought it gives you a framework provided way to handle required/optional params to a controller.

**Comment by lmcglohon** [comment]
@mejackreed @peetucket Is this OK to merge?

**Comment by peetucket** [comment]
Jack is correct that this is not optimal, but since there is no underlying model object against which we are checking required params, I'm not sure how to make this much cleaner.  There were no checks before at all.

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (cbeer) [inline app/controllers/items_controller.rb]: Strong params is enforced at the controller layer, e.g.

```
def record_params
  params.require(:type)
  params.require(:druid)
  params.require(:targets)
  params.slice(:type, :druid, :tarets)
end
```

Comment (peetucket) [inline app/controllers/items_controller.rb]: Does this work when you aren't using the params in a model object though?  I thought it had do with mass assignment and not when you are just using the values to pass to a method.

Comment (cbeer) [inline app/controllers/items_controller.rb]: Yep; mass assignment is just one of the beneficiaries of this API.

Comment (jkeck) [inline app/controllers/items_controller.rb]: This is a pretty useful overview: http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters

Comment (peetucket) [inline app/controllers/items_controller.rb]: oh cool, see my latest push...is that what you were thinking about?

Comment (mejackreed) [inline app/controllers/items_controller.rb]: I'm pretty sure you can here. The params don't need to be directly related to a model. 

From http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters

> In addition, parameters can be marked as required and will flow through a predefined raise/rescue flow that will result in a 400 Bad Request being returned if not all required parameters are passed in.

**Pull request merged**